src/components/directory-item-selector.js

```diff
@@ -64,12 +65,22 @@ const DirectoryItemSelector = (props) => {
         });
     };
 
+    function sortHandlingDirectories(a, b) {
+        if (a.type === elementType.DIRECTORY && a.type !== b.type) {
```

Comment:
Add a comment that this code is needed to put nodes with children (aka directories) first then other types of nodes (leaves) alphabetically sorted.

Reply:
Done